profiles/base.yaml

```diff
@@ -288,6 +288,8 @@ safe_room_empaths:
 # Spellcasting (and Khri) settings
 use_harness_when_arcana_locked: false
 combat_trainer_buffs_force_cambrinth: true
+# always skip cambrinth and use harness for non-buffs in combat-trainer.
```

Comment:
❓ Is this description correct? If I'm reading this right, as written this won't 'always skip cambrinth and use harness'. It will force `@harness_for_attunement` to `true`, but that still gets passed into `check_to_harness` which will still check Attunement vs Arcana exp.